superduperdb/jobs/job.py

```diff
@@ -92,15 +98,17 @@ class FunctionJob(Job):
     :param callable: function to be called
     :param args: positional arguments to be passed to the function
     :param kwargs: keyword arguments to be passed to the function
+    :param compute_kwargs: Arguments to use for model predict computation
```

Comment:
Please add an example somewhere of how we can use this `compute_kwargs`.

Reply:
done